android/build.gradle

```diff
@@ -9,7 +9,7 @@ android {
         minSdkVersion 19
         targetSdkVersion 28
         versionCode 26
-        versionName "0.4.0"
+        versionName "0.5.0"
```

Comment:
I would wait to bump the version until we actually want to release a new version.

Reply:
Hmm, but it's a new version that then needs to be referenced by the Android TV build.

Reply:
I hadn't seen that, works for me. We should only bump to 0.4.1 though, especially since 1.0.0 should only happen when we can generate the client with OpenAPI.

Reply:
I would still recommend 0.5.0, since the smallest digit is for bugfixes only and there is no rule that you have to roll over from 0.9.0 to 1.0.0. As I understand it, we can keep incrementing the second number as far as we like. 

Otoh, everything with leading zero is by definition unstable anyway, so we might also go with your recommendation. 

@thornbill Thoughts on this?

Reply:
I'm not personally a fan of numbers past nine for minor versions but I suppost the Android TV app is already using them so it's irrelevant here.

Reply:
I get that it doesn't look that nice aesthetically, but I think it is the correct thing to do according to https://semver.org/